docs/api.md

```diff
@@ -460,12 +460,12 @@ JSON post body can be sent in the following format:
 {
   "streams": [
     {
-      "labels": "<LogQL label key-value pairs>",
-      "entries": [
-        {
-          "ts": "<RFC3339Nano string>",
-          "line": "<log line>"
-        }
+      "stream": {
+        "label": "value"
+      },
+      "values": [
+          [ "<unix epoch in nanoseconds>", "<log line>" ],
+          [ "<unix epoch in nanoseconds>", "<log line>" ]
```

Comment:
This removes the documentation for the old format; can you add the `/api/prom/push` docs in the same way I do with the other old API paths?

Reply:
Good catch.  Added.